src/_includes/core-nodes/udp-out-use-case.md

```diff
@@ -0,0 +1,49 @@
+## What is udp-out node in Node-RED?
+
+The udp-out node in Node-RED is a node used for sending UDP messages to a specified network destination. When you add a udp-out node to your Node-RED flow, you configure it with the IP address and port of the destination device or service. Then, any messages received by the udp-out node are sent as UDP packets to that destination. This node is particularly useful for applications where real-time communication or lightweight message transmission is required.
+
+### What is Udp?
```

Comment:
UDP, and link this time to the UDP-in. Don't repeat content, SEO / Google doesn't like that.